docs/source/getting_started/debugging.rst

```diff
@@ -12,6 +16,10 @@ When an vLLM instance hangs or crashes, it is very difficult to debug the issue.
 - **Loading the model from disk**: If the model is large, it can take a long time to load the model from disk. Please take care of the location you store the model. Some clusters have shared filesystems across nodes, e.g. distributed filesystem or network filesystem, which can be slow. It would be better to store the model in a local disk. In addition, please also watch the CPU memory usage. When the model is too large, it might take much CPU memory, which can slow down the operating system because it needs to frequently swap memory between the disk and the memory.
 - **Tensor parallel inference**: If the model is too large to fit in a single GPU, you might want to use tensor parallelism to split the model across multiple GPUs. In that case, every process will read the whole model and split it into chunks, which makes the disk reading time even longer (proportional to the size of tensor parallelism). You can convert the model checkpoint to a sharded checkpoint using `the provided script <https://docs.vllm.ai/en/latest/getting_started/examples/save_sharded_state.html>`_ . The conversion process might take some time, but later you can load the sharded checkpoint much faster. The model loading time should remain constant regardless of the size of tensor parallelism.
 
+Note, over mainstream platforms and models (<=70B), it should not take more than **one hour** to load the model, usually just several minutes. So if you have wait for more than one hour, it is likely that the vLLM instance is stuck somewhere.
+
+One notorious crash issue is CUDA OOM (Out-of-Memory) error. If you encounter this issue, please refer to :ref:`CUDA OOM Section (Keep Updating) <cuda-oom>`.
```

Comment:
I won't classify OOM as crash.

Reply:
Thanks for the comment! I personally agree it can be a bit weird to call OOM a crash. However, this leads to a problem of what can be called a "crash" -- I think for general users, they see a (lengthy) stacktrace and they feel this might be a crash. I know this definition is too broad and can get us into trouble to include too many troubleshooting sections. I am happy and open to revising accordingly -- maybe I should remove "crash" in this sentence?